apps/cli/rollback.py

```
@@ -0,0 +1,272 @@
+        try:
+            counts = db_rollback_session(
+                sid,
+                dry_run=args.dry_run,
+                scope=args.scope,
+                force=args.force,
+            )
+        except ValueError as e:
+            logger.error("Refused to roll back session %s: %s", sid, e)
+            close_db()
+            return 2
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)